rfcs/shared/build-system/07-standard-package-structure-for-v9.md

```diff
@@ -22,12 +22,26 @@ The proposed folder organization can be seen below:
 |- stories/
   |- assets/
   |- {componentName}/ //story files
-|- e2e/
-  |- assets/
-  |- e2e tests
 |- src/
-  |- common/
-  |- components/ //implementation and test files
+  |- components/
+    |- {ComponentName}/ //implementation, unit and cypress test files
+      |- index.ts
+      |- {ComponentName}.tsx
+      |- {ComponentName}.types.ts
+      |- {ComponentName}.test.tsx
+      |- {ComponentName}.e2e.tsx
```

Comment:
**thought:** i have mixed feelings about this. while I understand the reasoning because we have all the context, it might feel missleading to OSS contributors what `e2e` actually means. 

Nx uses `cy` suffix, which explicitly express the intent `{ComponentName}.cy.tsx` (cypress component testing)

One downside of this is if we gonna switch to something else in the future, we would need to change the suffix ( lets say we switch to  playwright ).

WDYT?

Reply:
I'm in favor of being more explicit and using `.cy.`